tmail-backend/jmap/extensions/src/test/scala/com/linagora/tmail/james/jmap/json/LongLivedTokenSerializerTest.scala

```diff
@@ -48,7 +48,9 @@ class LongLivedTokenSerializerTest {
 
     val longLivedTokenSetResponse: LongLivedTokenSetResponse = LongLivedTokenSetResponse(ACCOUNT_ID,
       Some(created),
-      Some(notCreated))
+      None,
+      Some(notCreated),
+      None)
```

Comment:
Just seeing this... I guess would be better to have a full serialization for the test here?